components/GlobalStyles.js

```diff
@@ -51,15 +55,89 @@ mark { background-color: white;
 
 }
 li mark {
+    
     background-color: rosybrown;
 }
 
     a {
   text-decoration: none;
+  color: black;
+}
+
+.swiper-pagination {
+    
+
+
+}
+.document{
+   padding:2%;
+   background-color: #D0B3B3;
+   
+}
+
+.quote-card {
+    background-color: white;
+    padding: 1rem;
 
 
+}  
+.quote {font-family: SommerwerkInk;
+    font-size: 1.5rem;
+    
+}
+
+.quote-source {font-family: 'Trade Gothic';
+    font-size: 1rem;
+    
+}  
+.figcaption-ancient{
+    font-family: SommerwerkInk;
+    font-size: 1.5rem;
+    padding: 1rem;
+    color: #D0B3B3;
+}
+.credits-card{
+    background-color: white;
+    padding: 1rem;
+
+
+}  
+
+.mySwiper,
+.mySwiper2 {
+     height: 90vh;
+     width: 90vw;
+     
+     
+     
+
+}
+    
+
+.swiper-pagination-bullet{
+    background-color: #D0B3B3 !important;
+    opacity: 80%;
+    z-index: 999999;
+    height: 1rem;
+    width: 1rem;
+    
+    
+}
+.swiper-pagination-bullet-active{
+    background-color:  grey !important;
+    opacity: 80%;
+    z-index: 999999;
+    height: 1rem;
+    width: 1rem;
 }
 
+.swiper-pagination-vertical {
+    
+    
+    z-index: 99999
+    }
+    
+    
```

Comment:
It looks like prettier is not working, your blank lines are all over the place ;)